src/modules/Support/Service.php

```diff
@@ -798,8 +798,6 @@ public function ticketCreateForAdmin(\Model_Client $client, \Model_SupportHelpde
 
     public function ticketCreateForGuest($data)
     {
-        $this->di['validator']->isEmailValid($data['email']);
```

Comment:
We really *have* to do input validation here. Both `ticketCreateForGuest` and `publicTicketCreate` are used to create tickets for non-clients, so the email param is coming from an un-trusted source.

Reply:
Agree - but `isEmailValid` no longer exists. Perhaps reworking around `FOSSBilling\Tools` `validateAndSanitizeEmail()`?

Reply:
Yeah, that'll work great to perform the validation / sanitation needed here

Reply:
Updated to use `Tools` email validation function, including the related tests. 🙂